source/dem/read_mesh.cc

```diff
@@ -0,0 +1,66 @@
+#include <dem/read_mesh.h>
+
+template <int dim>
+void
+read_mesh(const DEMSolverParameters<dim> &           parameters,
+          const ConditionalOStream &                 pcout,
+          parallel::distributed::Triangulation<dim> &triangulation,
+          double &triangulation_cell_diameter)
```

Comment:
How is this function different than the read mesh function that is used in the core lethe library?